ruleset/phpstan.neon

```diff
@@ -1,10 +1,11 @@
 parameters:
-    level: max
+    level: 8
```

Comment:
I would suggest to you to leave `max` here, and fix errors or generate a baseline :wink:

Reply:
Job done ;)